app/views/layouts/_nav.html.erb

```diff
@@ -65,16 +65,34 @@
 		</ul>
 	</div>
   <% if @deployment_notification.present?%>
-    <% if Time.zone.now >= @deployment_notification.display_time %>
+    <% if Time.zone.now >= @deployment_notification.display_time%>
     <div class="notification-banner">
-      <div class="col-sm-3"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+      <button type="button" class="close" id='close-notification-banner' aria-label="Close" style="padding-right: 10px;">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <div class="col-sm-3">
+        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
         <span>Service Alert: Starting on</span></br>
-        <span><%= @deployment_notification.deployment_time.strftime("%a %b %d, %Y %I:%M%p") %> ET</span>
+        <span><%= @deployment_notification.deployment_time.strftime("%a %b %d, %Y %I:%M %p") %> ET</span>
       </div>
-      <div>
-        <div class="align-middle"><%= @deployment_notification.message %></div>
+      <div >
+        <span class="align-middle"><%= @deployment_notification.message %></span>
       </div>
+
     </div>
     <%end %>
+    <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+        $('#close-notification-banner').click(function() {
+            $(".notification-banner").hide();
+            if ( typeof Cookies.get('hide_deployment_notification') === 'undefined' ) {
```

Comment:
We'll need to ensure that hiding one deployment's banner doesn't hide the next deployment's banner.  

Deleting the `hide_deployment_notification` cookie upon creating a banner or including a string representation of `@deployment_notification.deployment_time` in the cookie name seem like two ways to achieve that.

Reply:
We discussed this, and the plan is to use the ID of the deployment notification as the value for the cookie.  That's the simplest check to make (no date math required).

Reply:
Excellent!